src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
move Random to class field

Reply:
Is this the best solution? I can't find why.

Reply:
same as here https://github.com/mate-academy/jv-lottery/pull/423#discussion_r942357870